index.cjs

```diff
@@ -753,19 +753,40 @@ safeql: try {
   }
 
   // Abort early if either of these modules are not installed
-  require.resolve('@ts-safeql/eslint-plugin');
-  require.resolve('dotenv-safe');
+  try {
+    require.resolve('@ts-safeql/eslint-plugin');
+    require.resolve('dotenv-safe');
+  } catch (error) {
+    throw new Error(
+      `SafeQL configuration failed
+
+Please reinstall the UpLeveled ESLint Config using the instructions on https://www.npmjs.com/package/eslint-config-upleveled
+
+${
+  typeof error === 'object' && error !== null && 'message' in error
+    ? error.message
+    : ''
+}
+    `,
+    );
+  }
 
   // @ts-ignore 2307 (module not found) -- The require.resolve() above will ensure that dotenv-safe is available before this line by throwing if it is not available
   require('dotenv-safe').config();
 
-  if (
-    !process.env.PGHOST ||
-    !process.env.PGUSERNAME ||
-    !process.env.PGPASSWORD ||
-    !process.env.PGDATABASE
-  ) {
-    throw new Error('Environment variables are not set');
+  const requiredEnvVars = ['PGHOST', 'PGUSERNAME', 'PGPASSWORD', 'PGDATABASE'];
```

Comment:
we can inline this below, since we're using it only 1 time

Reply:
Updated, and inlined
[36f4ceb](https://github.com/upleveled/eslint-config-upleveled/pull/234/commits/36f4cebb041441710804bac044e387fb3208ce39)